feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -49,6 +54,37 @@ class IdentityManager(
         }
     }
 
+    fun logIn(
+        newAppUserID: String,
+        onSuccess: (PurchaserInfo, Boolean) -> Unit,
+        onError: (PurchasesError) -> Unit
+    ) {
+        if (newAppUserID.isBlank()) {
+            onError(PurchasesError(PurchasesErrorCode.InvalidAppUserIdError,
+                "appUserID can't be null, empty or blank").also { errorLog(it) })
```

Comment:
Should we log here or in `Purchases.logIn`?

Reply:
I think the string should be part of `IdentityStrings`

Reply:
I forgot to move this to IdentityStrings :facepalm: Added now. As for logging here or in `Purchases`, I'm honestly not sure which one's better. Logging seems to be sort of all over the place, where the success and info paths are logged in this file, but error cases can be anywhere from `Backend` to `Purchases`. I think a reasonable criteria would be "log at the layer at which the error happens". If so, that'd be here for this particular case. 
Thoughts?

Reply:
I feel the same. I like the idea of logging when it happens. I don't think we should log directly in the `errorLog` function, but yes when it happens. I think we have more context to log correctly in the spot where the error happens